drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4447,6 +4447,17 @@ void RasterizerSceneGLES3::render_scene(const Transform &p_cam_transform, const
 		glDisable(GL_BLEND);
 	}
 
+	if (env && env->bg_mode == VS::ENV_BG_SKY && (!storage->frame.current_rt || (!storage->frame.current_rt->flags[RasterizerStorage::RENDER_TARGET_TRANSPARENT] && state.debug_draw != VS::VIEWPORT_DEBUG_DRAW_OVERDRAW))) {
+
+		/*
+		if (use_mrt) {
+			glBindFramebuffer(GL_FRAMEBUFFER,storage->frame.current_rt->buffers.fbo); //switch to alpha fbo for sky, only diffuse/ambient matters
```

Comment:
Should this comment still be here?

Reply:
My gut says no. But GLES3 isnt going to be around for long, so im trying to change as little as possible.